docs/learning/tutorials/develop-desktop-viewer.md

```diff
@@ -1,36 +1,46 @@
-# Developing a desktop viewer
+# Developing a Desktop Viewer
 
 ## Setup
 
-- [Install necessary prerequisites]($docs/getting-started/development-prerequisites)
-- [Clone desktop-starter repo](https://github.com/imodeljs/desktop-starter)
+- [Install necessary prerequisites]($docs/learning/tutorials/development-prerequisites).
+- From a terminal, `npx create-react-app your-app-name --template @itwin/desktop-viewer --scripts-version @bentley/react-scripts`
+  - This will generate a new application based on the iTwin Viewer React component in the `your-app-name` directory.
+- Open the `your-app-name` directory in VS Code
+- Follow these steps to obtain a new OIDC client to use in your viewer application:
+  1. Go to <https://developer.bentley.com>
+  2. Click the **Sign In** button and sign-in using your Bentley account credentials
+      - If you have not already registered, click **Register now** and complete the registration process.
+  3. Navigate to the [My Apps](https://developer.bentley.com/my-apps/) page
+  4. Click the **Register New** button
+  5. Give your application a Name
+  6. Select the **Visualization** API
+  7. Select application type **Desktop**

```

Comment:
The "Register App" page has this type listed as "Desktop/Mobile"
![image](https://user-images.githubusercontent.com/38542573/130464616-954dd0cc-a4ef-4ad6-a266-4b95baab194f.png)

Reply:
Yea, I didn't really think it was too necessary to match it but switched it anyway to avoid any confusion.